site/_apps/react-hn-stephen.txt

```diff
@@ -1,5 +1,5 @@
 ---
-weight: 9
+weight: 14
```

Comment:
The new application weight (we've landed more apps since this was first submitted) is `23`. Can we update 14 to 23?